nodeup/pkg/model/kubeapiserver.go

```diff
@@ -70,6 +71,19 @@ func (b *KubeAPIServerBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
+	// @check if we are using secure client certificates for kubelet and grab the certificates
+	{
+		if b.UseSecureKubelet() {
```

Comment:
Indeed ... i'm not a fan of bracketed code either, but i wanted to stay consistent and everything else else bracketed, so i felt the peer pressure :-)